basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/UserFinderOrCreatorImpl.java

```diff
@@ -81,7 +86,19 @@ public User findOrCreateUser(Map payload, boolean trustedConsumer) throws LTIExc
                 throw new LTIException("launch.user.invalid", "user_id=" + user_id, e);
             }
 
-        } else {
+        }else
```

Comment:
else clause should be bracketed.  Mixing bracketed and unbracketed clauses is very likely to lead to bugs.